packages/salesforcedx-vscode-apex/README.md

```diff
@@ -35,6 +37,13 @@ The Problems view in the bottom pane also lists the syntax errors. Double-click
 The Apex outline view shows the structure of the Apex class or trigger that’s open in the editor. For a list of the symbols in your file, press Cmd+Shift+O (macOS) or Ctrl+Shift+O (Windows or Linux). To jump to one of the symbols, select it in the list.  
 ![Outline view, showing the symbols in an Apex class](https://raw.githubusercontent.com/forcedotcom/salesforcedx-vscode/develop/packages/salesforcedx-vscode-apex/images/apex_outline.png)
 
+## Enable Code Smartness for SObjects
+To activate this extension’s code smartness features for standard and custom objects and their fields, including for custom fields on standard objects, press Cmd+Shift+P (macOS) or Ctrl+Shift+P (Windows or Linux) and then select **SFDX: Refresh SObject Definitions** from the command palette. 
+
+When you refresh your sObject definitions, VS Code uses your default scratch org to generate faux Apex classes. These faux classes represent the standard and custom objects that the admin user of your default scratch org has access to. The classes are stored in a hidden directory on your local workstation. Don’t edit the faux classes! They are deleted and regenerated each time that you refresh your sObject definitions. To modify your sObjects, either modify the objects’ `.object-meta.xml` and `.field-meta.xml` files (and then run **SFDX: Push Source to Default Scratch Org**) or make changes declaratively in your scratch org (and then run **SFDX: Pull Source from Default Scratch Org**). The scratch org’s admin user doesn’t automatically gain access to new custom objects, so be sure to assign new permissions to the user as necessary. To assign permissions from the command line, run `sfdx force:user:permset:assign -n YourPermSetName`. 
```

Comment:
Similar to other comment re: stacked conjunctions - I would put a command before the OR. "Org**), or make changes declaratively..."